### PR TITLE
pw-cli: use a more clear description instead of a generic one

### PR DESCRIPTION
**Pull request by vitorhcl**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by cyqsimon — APPROVED** [review]
LGTM

File: pages.pt_BR/linux/pw-cli.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # pw-cli
 
-> A interface de linha de comando do PipeWire.
+> Gerencie módulos, objetos, nós, dispositivos, conexões e muito mais de uma instância PipeWire.
```
Comment (isaacvicente) [inline pages.pt_BR/linux/pw-cli.md:3]:
```suggestion
> Gerencia módulos, objetos, nós, dispositivos, conexões e muito mais de uma instância PipeWire.
```

**Review by isaacvicente — COMMENTED** [review]
Pequena alteração.

**Review by isaacvicente — APPROVED** [review]

**Pull request merged**